aptos-move/framework/aptos-stdlib/sources/signature.move

```diff
@@ -9,7 +9,16 @@
 ///     as per the [IETF BLS draft standard](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature#section-2.1).
 ///     TODO: Describe APIs
 module aptos_std::signature {
+    use std::bcs;
     use std::option::Option;
+    use aptos_std::type_info::{Self, TypeInfo};
+
+    // Proof a generic type that includes the type info of a Proof
+    // and the corresponding message of ProofType
+    struct Proof<ProofType> has drop {
```

Comment:
@davidiw, for now we'll leave this as is, but this `Proof` name should probably be replaced with something more accurate like `Message` or `Challenge`. This is not a proof in any sense of the word: a signature on such a struct would constitute a proof.

I'll rename this once I understand its usage in the next PR.